pkg/util/utils.go

```diff
@@ -313,33 +313,31 @@ func getTomlStorage(storeOptions *storage.StoreOptions) *tomlConfig {
 	return config
 }
 
-// GetDefaultStoreOptions returns the storage ops for containers
-func GetDefaultStoreOptions() (storage.StoreOptions, error) {
```

Comment:
I think the reason for having this function here is that `pkg/util` is vendored inside buildah and `GetDefaultStoreOptions` is used there as well.  Do you think it makes sense if we leave it?

Anyway, should not be too difficult to reimplement in Buildah if it is dropped

Reply:
+1 to a Buildah dependency on this.  Great catch @giuseppe